contrib/test_jenkins.sh

```diff
@@ -379,7 +379,7 @@ run_coverity() {
 		cov_build="$WORKSPACE/$cov_build_id"
 		rm -rf $cov_build
 		cov-build   --dir $cov_build $MAKE all
-		cov-analyze --dir $cov_build
+		cov-analyze $COV_OPT --dir $cov_build
```

Comment:
it's defined by Mellanox jenkins system. It's value is:
"--enable-fnptr --fnptr-models --all --paths 20000"